crates/core/src/yarn_fn/function_registry.rs

```diff
@@ -1,4 +1,6 @@
 use crate::prelude::*;
+#[cfg(feature = "bevy")]
+use bevy::prelude::*;
```

Comment:
Would be nice to run all our tests once with and once without bevy. Could you expand the CI yaml?

Reply:
[I don't think that worked](https://github.com/YarnSpinnerTool/YarnSpinner-Rust/actions/runs/14372879503/job/40299099001?pr=224#step:5:769). Looks like the Bevy feature is active because a downstream package of the workspace (namely the Bevy plugin) requires it. Maybe change the call to only test the `yarnspinner` crate?

Reply:
Ok. I think I've got all the tests running in the right places with the right sets of features now.

Reply:
Fingers crossed